content/doc/addons/postgresql/_index.md

```diff
@@ -18,42 +18,38 @@ aliases:
+Versions currently available for order are as follows:
+- On shared (DEV) cluster: PostgreSQL version 15
+- On dedicated plans: PostgreSQL versions 10, 11, 12, 13, 14 and 15
 
 {{< readfile file="db-backup.md" >}}
 
 ## Migrating from an old database
 
-Some applications require a populated database to run properly.
-If you want to import your **SQL** dump, you can use several methods:
-
-1. [Our WebGUI (Adminer)](https://dbms-adminer.clever-cloud.com/).
-2. Command line tool for PostgreSQL administration like `psql`.
-3. Any PostgreSQL client such as [pgAdmin](https://www.pgadmin.org/).
+Some applications require a non-empty database to run properly. If you want to import your **SQL** dump, you can use several methods:
+1. [WebGUI (Adminer)](https://dbms-adminer.clever-cloud.com/)
+2. Command line tool for PostgreSQL administration like `psql`
+3. Any PostgreSQL client such as [pgAdmin](https://www.pgadmin.org/)
 
 ## Direct access
 
-All our dedicated PostgreSQL databases are served via a proxy. To reduce the latency you can bypass this proxy by generating direct hostname and port for the addon. You can do it by clicking the "Generate direct hostname and port" on the addon dashboard.
-
-This action will add new environment variables to reach the addon without any proxy.
+A proxy serves all dedicated PostgreSQL databases. In some cases, this can add some latency between applications and their database. If this is an issue, you must generate a direct hostname and port for the add-on to bypass the proxy, using the "Generate direct hostname and port" button in the add-on dashboard.
 
-Keep in mind that usage of direct access is a trade-off: when you migrate your addon, you will need to to generate hostname and port again so your application must update these environment while proxy usage changes nothing.
+Generating direct access adds new variables to the add-on's environment, allowing connections without going through the proxy.
 
-## Encryption at rest
+{{< callout type="info" >}} 
+Direct access doesn't persist after upgrading/migrating a database. Manually re-generate direct host name and port after an upgrade to allow applications linked through direct access to connect to the database. 
 
-Encryption at rest is available on PostgreSQL. You can have more information on the [dedicated page]({{< ref "doc/administrate/encryption-at-rest.md" >}})
+Such a limitation doesn't exist when using proxy access.
+{{< /callout >}}
 
 ## Note on shared databases
 
-Free databases are instantiated from a shared cluster. The way postgresql works on shared clusters is that a "pg_database" table references the name, and only the name, of other databases in that same cluster. The data, on the other hand, is not accessible.
+Encryption at rest is available on PostgreSQL. You can find more information on the [dedicated page]({{< ref "doc/administrate/encryption-at-rest.md" >}})
 
 This referencing does not exist for dedicated databases.
```

Comment:
Something went horribly wrong here it seems. The entire section on the shared cluster is gone.